exercises/pangram/canonical-data.json

```diff
@@ -1,68 +1,87 @@
   ],
-  "version": "1.1.0",
+  "version": "1.2.0",
   "cases": [
     {
-      "description": "Check if the given string is an pangram",
+      "description": "checks if the given string is a pangram.",
       "comments": [
         "Output should be a boolean denoting if the string is a pangram or not."
       ],
       "cases": [
         {
-          "description": "sentence empty",
+          "description": "returns false for an empty message.",
           "property": "isPangram",
           "input": "",
           "expected": false
         },
         {
-          "description": "pangram with only lower case",
+          "description": "recognizes a perfect lower case pangram.",
           "property": "isPangram",
-          "input": "the quick brown fox jumps over the lazy dog",
+          "input": "abcdefghijklmnopqrstuvwxyz",
           "expected": true
         },
         {
-          "description": "missing character 'x'",
+          "description": "recognizes lower case pangram with spaces.",
+          "property": "isPangram",
+          "input": "few quips galvanized the mock jury box",
+          "expected": true
+        },
+        {
+          "description": "recognizes missing characters, e.g. 'x'.",
           "property": "isPangram",
           "input": "a quick movement of the enemy will jeopardize five gunboats",
           "expected": false
         },
         {
-          "description": "another missing character 'x'",
+          "description": "recognizes missing characters, e.g. 'h'.",
           "property": "isPangram",
-          "input": "the quick brown fish jumps over the lazy dog",
+          "input": "five boxing wizards jump quickly at it",
           "expected": false
         },
         {
-          "description": "pangram with underscores",
+          "description": "recognizes an upper-case pangram.",
           "property": "isPangram",
-          "input": "the_quick_brown_fox_jumps_over_the_lazy_dog",
+          "input": "THE QUICK BROWN FOX JUMPS OVER THE LAZY DOG",
           "expected": true
         },
         {
-          "description": "pangram with numbers",
+          "description": "recognizes a missing character in upper-case, e.g. 'G'.",
+          "property": "isPangram",
+          "input": "THE QUICK BROWN FOX JUMPS OVER THE LAZY DOLL",
+          "expected": false
+        },
+        {
+          "description": "recognizes a mixed-case pangram.",
           "property": "isPangram",
-          "input": "the 1 quick brown fox jumps over the 2 lazy dogs",
+          "input": "ThE FiVe bOxInG WiZaRdS JuMp qUiCkLy",
           "expected": true
         },
         {
-          "description": "missing letters replaced by numbers",
+          "description": "recognizes a missing character in mixed-case, e.g. 'Y'.",
           "property": "isPangram",
-          "input": "7h3 qu1ck brown fox jumps ov3r 7h3 lazy dog",
```

Comment:
one interesting thing about this input is that the number of unique non-space characters in it is exactly 26, which is the intent of https://github.com/exercism/problem-specifications/pull/852 . 

Now it looks like we are testing mixed-case before non-alphabetics, all right, so we wouldn't have the letters -> numbers case, that seems fine.

If you believe that is still a useful bug to catch, then it would be useful to have a case like "Aabcdefghijklmnopqrstuvwxy"

Reply:
The criterion is not met by "ThE FiVe bOxInG WiZaRdS JuMp qUiCkL" because the number of unique non-space characters in it is 28.

Reply:
@petertseng Thanks for this hint. 
However, I am not sure I understand the intention of this bugfix or the changed testcase. Is it to test for mixed-case, but with 26 distinct chars? 
If so, I think we should add a testcase with exactly that intent to prevent future misunderstandings/changes.

Reply:
> Is it to test for mixed-case, but with 26 distinct chars?

Yes. The test case that got replaced is: lowercase + numbers, with 26 distinct non-space chars.

I was thinking it would get replaced with: lowercase + uppercase, with 26 distinct non-space chars.

I think you understood correctly.